teamworkApp/lib/dbCalls.py

```diff
@@ -0,0 +1,109 @@
+# muddersOnRails()
+# Sara McAllister November 5, 2017
+# Last updated: 11-8-2017
+
+# library for SQLite database calls for teamwork analysis app
+
+import contextlib
+import datetime
+import sqlite3
+
+DB = 'db/development.sqlite3'
+test_DB = 'db/test.sqlite3'
+
+def connect(sqlite_file):
+    """ Make connection to an SQLite database file """
+    conn = sqlite3.connect(sqlite_file)
+    c = conn.cursor()
+    return conn, c
+
+def close(conn):
+    """ Commit changes and close connection to the database """
+    conn.commit()
+    conn.close()
+
+@contextlib.contextmanager
+def dbconnect(test):
+    """Create cursor and ensure that DB connection closes"""
+    sqlite_file = DB
+    if test: sqlite_file = test_DB
+
+    conn, cursor = connect(sqlite_file)
+    try:
+        yield cursor
+    finally:
+        close(conn)
+
+def get_all_styles(test=False):
```

Comment:
Maybe add some big comments that delineate the `get` functions and the `insert` functions to make a quick skim of the file a bit easier to see where to find the functions you want?